config/application.rb

```diff
@@ -125,6 +126,7 @@ class Application < Rails::Application
 
     # StateFile
     config.state_file_start_of_open_intake = et.parse('2025-01-15 00:00:00')
+    config.state_file_tax_deadline = et.parse('2025-04-15 23:59:59')
```

Comment:
we used to call the general tax deadline date for statefile but these need to be separated now that statefile isn't moving forward here